packages/playground/src/components/webhooks/create-webhooks-form-fields.tsx

```diff
@@ -0,0 +1,166 @@
+import React from 'react'
+import { Input, Textarea, Checkbox, StackedList, Switch, RadioGroup, RadioGroupItem } from '@harnessio/canary'
+import { FormFieldSet, MessageTheme } from '../../index'
+import { WebhookEvent, FieldProps, BranchEvents, TagEvents, PREvents, EventTypes } from './types'
+
+export const WebhookToggleField: React.FC<FieldProps> = ({ register, watch, setValue }) => (
+  <StackedList.Root className="border-none">
+    <StackedList.Item disableHover isHeader>
+      <StackedList.Field
+        title="Enable the webhook"
+        description="We will deliver event details when this hook is triggered"
+      />
+      <StackedList.Field
+        label
+        secondary
+        title={
+          <div className="flex gap-1.5 items-center justify-end cursor-pointer">
+            <Switch
+              {...register!('state')}
+              checked={watch!('state')}
+              onCheckedChange={() => setValue!('state', !watch!('state'))}
+            />
+          </div>
+        }
+        right
+      />
+    </StackedList.Item>
+  </StackedList.Root>
+)
+
+export const WebhookNameField: React.FC<FieldProps & { disabled: boolean }> = ({ register, errors, disabled }) => (
```

Comment:
why do we need so much boilerplate for all the form fields? are we not using the schema framework which can render basic type inputs out-of-the-box?
cc @srdjan-harness